File: introduction/.py
import requests
from bs4 import BeautifulSoup

def scrape():
    url = 'https://www.example.com'
    try:
        response = requests.get(url)
        soup = BeautifulSoup(response.text, 'html.parser')
        title = soup.select_one('h1').text
        text = soup.select_one('p').text
        link = soup.select_one('a').get('href')
        print(title)
        print(text)
        print(link)
    except:
        print("Please check your connection")
    # except Exception as e:
    #     print(e)

if __name__ == '__main__':
    scrape()

File: introduction/.js
import * as cheerio from 'cheerio'

const url = 'https://www.example.com'

try {
    const response = await fetch(url)

    const $ = cheerio.load(await response.text())
    const title = $('h1').text()
    const text = $('p').text()
    const link = $('a').attr('href')
    console.log(title)
    console.log(text)
    console.log(link)
} catch {
    console.log("check your internet connection")
}
//} catch (e) {
//    console.log(e)
//}
